src/components/Navigation.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
+import styled from "styled-components";
 
 const Navigation = () => {
   return (
-    <ul className="navigation">
+    <StyledNavigation >
       <li>
-        <a href="#">Homepage</a>
+        <StyledLink href="#">Homepage</StyledLink>
       </li>
       <li>
-        <a href="#">Favorites</a>
+        <StyledLink href="#">Favorites</StyledLink>
       </li>
       <li>
-        <a href="#">Ramdom</a>
+        <StyledLink href="#">Random</StyledLink>
       </li>
-    </ul>
+    </StyledNavigation>
   );
 };
 
+
+const StyledNavigation = styled.div`
```

Comment:
This should definitely an `ul` element. Because you are using `li` elements here.